src/Transport.php

```diff
@@ -27,7 +27,10 @@ public function send(\Swift_Mime_SimpleMessage $message, &$failedRecipients = nu
         $this->beforeSendPerformed($message);
         $token = $this->getToken();
         $emailMessage = $this->getMessage($message);
-        $url = sprintf('https://graph.microsoft.com/v1.0/users/%s/sendMail', urlencode($emailMessage['from']['emailAddress']['address']));
+        $url = sprintf(
+            'https://graph.microsoft.com/v1.0/users/%s/sendMail',
+            urlencode(isset($this->config['username']) ? $this->config['username'] : $emailMessage['from']['emailAddress']['address'])
```

Comment:
urlencode should be on the right side of `?` 😆

Reply:
I don't think so? Whether we use the username or the address we have to encode it

Reply:
I seeeeee.... I missed that it's wrapping the whole ternary statement 🤦‍♂️